### ISSUE TITLE
End/Home keys do not work

### ISSUE THREAD

**Issue by Seldaek** [Alexpux/MSYS2-packages#396]
I just created a new MSYS2-based setup here, which I use together with latest ConEmu and run msys's bash in it.

I got everything to work pretty well except for the Home and End key. If I press them they just print `~` and do nothing.

If figured that I can fix it in `~/.inputrc` with this config:

```
"\e[1~": beginning-of-line         # Home
"\e[4~": end-of-line               # End
```

That makes them work locally, but once I am ssh'd in a machine they go back to not working. They worked fine with my old mingw+msys1 setup, including in ssh, so I am thinking it must be possible to fix but I have no clue how. Help would be appreciated if it's a failure on my part.

**Comment by maehne** [comment]
Have you tried with the ConEmu version from the msys/conemu-git package? With this version, HOME and END keys work in MSYS2 Bash on the command line. Though, I haven't figured yet out how to get more complex console applications such as vim or mc to work properly in ConEmu. Also, the clear command isn't working as expected and I had to emulate it via a bash alias echoing an escape sequence.

**Comment by Seldaek** [comment]
Yeah sorry the home/end was due to a misconfiguration of some sort. I figured it out and now those work fine even without inputrc tweaks.

I however have an issue with function keys and alt-1, alt-2, etc not working in irssi/screen. Not sure what's up there, but I am not sure which part is at fault in the whole chain.
